scripts/dev_scripts/integration_tests.sh

```diff
@@ -257,17 +257,8 @@ $RUN_MACARON analyze -c $WORKSPACE/tests/dependency_analyzer/configurations/micr
 
 check_or_update_expected_output $COMPARE_DEPS $DEP_RESULT $DEP_EXPECTED || log_fail
 
-# TODO: uncomment the test below after resolving https://github.com/oracle/macaron/issues/60.
-# echo -e "\n----------------------------------------------------------------------------------"
-# echo "micronaut-projects/micronaut-test: Check the resolved dependency output with config for cyclonedx gradle plugin (default)."
-# echo -e "----------------------------------------------------------------------------------\n"
-# DEP_EXPECTED=$WORKSPACE/tests/dependency_analyzer/expected_results/cyclonedx_micronaut-projects_micronaut-test.json
-# $RUN_MACARON analyze -c $WORKSPACE/tests/dependency_analyzer/configurations/micronaut_test_config.yaml || log_fail
-
-# python $COMPARE_DEPS $DEP_RESULT $DEP_EXPECTED || log_fail
-
 echo -e "\n----------------------------------------------------------------------------------"
-echo "micronaut-projects/micronaut-test: Check the e2e output JSON file with config and no dependency analyzing."
+echo "micronaut-projects/micronaut-test: Check the e2e output JSON file with config when automatic dependency resolution is skipped."
```

Comment:
If this section only check the output files without running any Macaron command ( the Macaron command run is above it), perhaps we should merge this comment with the one above it so that it appears to be one test case.

Reply:
No, these are two different log messages for separate comparisons.